app/charts/index.ts

```diff
@@ -287,7 +287,10 @@ export const getInitialConfig = ({
             componentIri: geoCoordinates[0]?.iri || geoShapes[0]?.iri || "",
             measureIri: measures[0].iri,
             hierarchyLevel: 1,
-            color: "#1f77b4",
+            colors: {
+              type: "single",
+              value: "#1f77b4",
+            },
```

Comment:
This is not a refactor since it's changing the schema. It should be more prominent from the commit title + description that this commit is "dangerous".